beluga_example/models/turtlebot.model.yaml

```diff
@@ -17,7 +17,7 @@ plugins:
     odom_frame_id: odom
     pub_rate: 10
     enable_odom_pub: true
-    odom_pose_noise: [0.005, 0.005, 0.01]
+    odom_pose_noise: [0.0002, 0.0002, 0.0001]
```

Comment:
@nahueespinosa meta: I'm a bit concerned about the order of magnitude. It's one thing for this to be an artifact of the simulation (ie. unrealistic noise PSD), it's a different thing if Beluga needs very precise odometry to function. Not to be addressed here though.

Reply:
> It's a different thing if Beluga needs very precise odometry to function.

I don't think that's the case. FWIW nav2_amcl has the same problem with this set of parameters.